crates/matrix-sdk/src/client/mod.rs

```diff
@@ -2519,6 +2519,16 @@ impl Client {
         let request = set_pusher::v3::Request::post(pusher);
         self.send(request, None).await
     }
+
+    /// Adds the specified user_id to the current account data's ignore_list
+    pub async fn ignore_user(&self, user_id: &OwnedUserId) -> Result<()> {
+        self.account().ignore_user(user_id).await
+    }
+
+    /// Removes the specified user_id to the current account data's ignore_list
+    pub async fn unignore_user(&self, user_id: &OwnedUserId) -> Result<()> {
+        self.account().unignore_user(user_id).await
+    }
```

Comment:
Why add these functions in both `Account` and `Client`?

Reply:
just as an helper since I wanted to add in FFI the function bot in Client and in RoomMember but since the account functions are pub(crate) I also needed to expose them in the Client

Reply:
Hm, I'd rather remove the ones in `Client` and make these public. I guess these could also be attached to the `RoomMember` but given recent discussion about potential confusion in that, let's just keep it here for now, and make it public since it's needed for FFI.

Reply:
done